### ISSUE TITLE
XML and OVMF file for Windows VM on SEV?

### ISSUE THREAD

**Issue by FishmanL** [AMDESE/AMDSEV#209]
Hello -- I'm trying to use this repo to launch a windows guest VM instead of a linux VM, and am running into several issues -- as best I can tell, the UEFI variables are misset somehow, so the VM is failing to boot (it boots fine outside SEV on the same host).  Has anyone managed to find a working XML/OVMF setup for windows VMs?  (Server 2019)

**Comment by tlendacky** [comment]
You're trying to run Windows with SEV enabled? Windows isn't enlightened to run under SEV as far as I'm aware, so I wouldn't expect it to be able to boot.

**Comment by FishmanL** [comment]
Wait, but Azure runs windows VMs under SEV

**Comment by tlendacky** [comment]
Yes, they use a paravisor, It is one of the goals of coconut-svsm, but not yet implemented, to be able to run un-enlightened guest OSes.

**Comment by FishmanL** [comment]
Ahhh, I see, thanks
